packaging/linux/buildpackage.sh

```diff
@@ -48,9 +48,9 @@ fi
 # Add native libraries
 echo "Downloading appimagetool"
 if command -v curl >/dev/null 2>&1; then
-	curl -s -L -O https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage
+	curl -s -L -o $HOME/appimagetool-x86_64.AppImage https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage
 else
-	wget -cq https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage
+	wget -cq -O $HOME/appimagetool-x86_64.AppImage https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage
```

Comment:
Undo all those changes where you hard-code `$HOME`.

Reply:
This is required to prevent the packaging scripts from downloading a file into the source tree, thus meaning that running them results in changes to the repository. It specifically does not hard-code `$HOME`: it uses the value of that variable whatever it may be.